src/index.html

```diff
@@ -1,162 +1,210 @@
-<link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
-<link rel="manifest" href="/favicons/site.webmanifest">
-<link rel="mask-icon" href="/favicons/safari-pinned-tab.svg" color="#38b5ad">
-<link rel="shortcut icon" href="/favicons/favicon.ico">
-<meta name="msapplication-TileColor" content="#ffffff">
-<meta name="msapplication-TileImage" content="/favicons/mstile-144x144.png">
-<meta name="msapplication-config" content="/favicons/browserconfig.xml">
-<meta name="theme-color" content="#ffffff">
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/favicons/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/favicons/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="194x194"
+      href="/favicons/favicon-194x194.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="/favicons/android-chrome-192x192.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/favicons/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/favicons/site.webmanifest" />
+    <link
+      rel="mask-icon"
+      href="/favicons/safari-pinned-tab.svg"
+      color="#38b5ad"
+    />
+    <link rel="shortcut icon" href="/favicons/favicon.ico" />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta
+      name="msapplication-TileImage"
+      content="/favicons/mstile-144x144.png"
+    />
+    <meta name="msapplication-config" content="/favicons/browserconfig.xml" />
+    <meta name="theme-color" content="#ffffff" />
 
-<style type="text/css">
-#loading {
-	position: absolute;
-	top: 0px;
-	left: 0px;
-	text-align: center;
-	width: 100%;
-	height: 100%;
-	z-index: 1100;
- 	background-color: rgba(250,250,250,0.4);
-}
-#loading div {
-	position: absolute;
-	top: 40%;
-	width: 100%;
-	font-family: Arial,serif;
-	font-size: 200%;
-	font-weight: bold;	
-} 
-</style>
-	<title>Feinstaub-Karte Luftdaten.info</title>
-</head>
-<body>
-	<div id="topcontainer">
-		<div id="betterplace"></div>
-		<div id="update"></div>
-	</div>
-	<span id="menu">&#9776;</span>
-	<div id="loading"><div>Loading data...<br/><span style="font-size:70%">Internet Explorer users: Please use the <a href="/old/">old version</a>.</span></div></div>
+    <style type="text/css">
+      #loading {
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        text-align: center;
+        width: 100%;
+        height: 100%;
+        z-index: 1100;
+        background-color: rgba(250, 250, 250, 0.4);
+      }
+      #loading div {
+        position: absolute;
+        top: 40%;
+        width: 100%;
+        font-family: Arial, serif;
+        font-size: 200%;
+        font-weight: bold;
+      }
+    </style>
+    <title>Sensors.AFRICA BASE</title>
```

Comment:
Why BASE? And why not just sensors.AFRICA?

Reply:
@kilemensi That was meant to be a placeholder. Just removed it